tests/test_product_deletion.py

```
@@ -0,0 +1,65 @@
+import pytest
+from app import app, db, Product
+import json
+import uuid
+
+@pytest.fixture(scope='module')
+def test_client():
+    app.config['TESTING'] = True
+    # Use an in-memory SQLite database for testing
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///:memory:'
+    app.config['WTF_CSRF_ENABLED'] = False
+    app.config['SECRET_KEY'] = 'test-secret-key' # Needed for session
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)